diff --git a/node_modules/@nuxt/content-theme-docs/src/components/app/AppFooter.vue b/node_modules/@nuxt/content-theme-docs/src/components/app/AppFooter.vue
index ca6b1a0..50890bf 100644
--- a/node_modules/@nuxt/content-theme-docs/src/components/app/AppFooter.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/components/app/AppFooter.vue
@@ -2,12 +2,12 @@
   <footer class="h-16 relative w-full border-t dark:border-gray-800 bg-white dark:bg-gray-900 z-20">
     <div class="h-full w-full container mx-auto px-4 lg:px-8 flex items-center justify-between">
       <div class="flex items-end">
-        <a href="https://nuxtjs.org" target="_blank" rel="noopener">
+        <!-- <a href="https://nuxtjs.org" target="_blank" rel="noopener">
           <BuiltWithNuxtLight class="h-4 light-img" />
           <BuiltWithNuxtDark class="h-4 dark-img" />
-        </a>
+        </a> -->
       </div>
-      <div class="flex items-center space-x-4">
+      <div class="flex items-center space-s-4">
         <AppLangSwitcher />
         <AppColorSwitcher />
       </div>
diff --git a/node_modules/@nuxt/content-theme-docs/src/components/app/AppHeader.vue b/node_modules/@nuxt/content-theme-docs/src/components/app/AppHeader.vue
index 14c771e..ac831f8 100644
--- a/node_modules/@nuxt/content-theme-docs/src/components/app/AppHeader.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/components/app/AppHeader.vue
@@ -6,7 +6,7 @@
   >
     <div class="container mx-auto flex-1 px-4 lg:px-8">
       <div class="flex items-center justify-between h-16">
-        <div class="lg:w-1/5 flex items-center pr-4" @click.stop="noop">
+        <div class="lg:w-1/5 flex items-center pe-4" @click.stop="noop">
           <NuxtLink
             :to="localePath('/')"
             class="flex-shrink-0 flex-1 font-bold text-xl"
@@ -28,13 +28,13 @@
           <AppSearch v-else class="hidden lg:block" />
         </div>
         <div
-          class="lg:w-1/5 flex items-center pl-4 lg:pl-8"
+          class="lg:w-1/5 flex items-center ps-4 lg:ps-8"
           :class="{ 'justify-between': lastRelease && settings.layout !== 'single', 'justify-end': !lastRelease || settings.layout === 'single' }"
         >
           <NuxtLink
             v-if="lastRelease"
             :to="localePath('/releases')"
-            class="font-semibold leading-none text-gray-700 dark:text-gray-300 hover:text-primary-500 dark-hover:text-primary-500 text-base mr-4"
+            class="font-semibold leading-none text-gray-700 dark:text-gray-300 hover:text-primary-500 dark-hover:text-primary-500 text-base me-4"
             exact-active-class="text-primary-500"
           >{{ lastRelease.name }}</NuxtLink>
           <div class="flex items-center">
@@ -45,7 +45,7 @@
               rel="noopener noreferrer"
               title="Twitter"
               name="Twitter"
-              class="text-gray-700 dark:text-gray-300 hover:text-primary-500 dark-hover:text-primary-500 ml-4"
+              class="text-gray-700 dark:text-gray-300 hover:text-primary-500 dark-hover:text-primary-500 ms-4"
               :class="{
                 'hidden lg:block': settings.layout !== 'single'
               }"
@@ -59,7 +59,7 @@
               rel="noopener noreferrer"
               title="Github"
               name="Github"
-              class="text-gray-700 dark:text-gray-300 hover:text-primary-500 dark-hover:text-primary-500 ml-4"
+              class="text-gray-700 dark:text-gray-300 hover:text-primary-500 dark-hover:text-primary-500 ms-4"
               :class="{
                 'hidden lg:block': settings.layout !== 'single'
               }"
@@ -69,7 +69,7 @@
 
             <button
               v-if="settings.layout !== 'single'"
-              class="lg:hidden p-2 rounded-md text-gray-700 dark:text-gray-300 focus:outline-none -mr-2"
+              class="lg:hidden p-2 rounded-md text-gray-700 dark:text-gray-300 focus:outline-none -me-2"
               aria-label="Menu"
               @click.stop="menu = !menu"
             >
diff --git a/node_modules/@nuxt/content-theme-docs/src/components/app/AppPrevNext.vue b/node_modules/@nuxt/content-theme-docs/src/components/app/AppPrevNext.vue
index 24045ee..78b6a20 100644
--- a/node_modules/@nuxt/content-theme-docs/src/components/app/AppPrevNext.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/components/app/AppPrevNext.vue
@@ -11,7 +11,7 @@
       :to="localePath(prev.to)"
       class="text-primary-500 font-bold hover:underline flex items-center justify-start"
     >
-      <IconArrowLeft class="w-4 h-4 mr-1 flex-shrink-0" />
+      <!-- <IconArrowLeft class="w-4 h-4 mr-1 flex-shrink-0" /> -->
       <span class="truncate">{{ prev.title }}</span>
     </NuxtLink>
     <span v-else>&nbsp;</span>
@@ -22,7 +22,7 @@
       class="text-primary-500 font-bold hover:underline flex items-center justify-end"
     >
       <span class="truncate">{{ next.title }}</span>
-      <IconArrowRight class="w-4 h-4 ml-1 flex-shrink-0" />
+      <!-- <IconArrowRight class="w-4 h-4 ml-1 flex-shrink-0" /> -->
     </NuxtLink>
     <span v-else>&nbsp;</span>
   </div>
diff --git a/node_modules/@nuxt/content-theme-docs/src/components/app/AppToc.vue b/node_modules/@nuxt/content-theme-docs/src/components/app/AppToc.vue
index 0bb9b57..6d3efad 100644
--- a/node_modules/@nuxt/content-theme-docs/src/components/app/AppToc.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/components/app/AppToc.vue
@@ -22,13 +22,13 @@
           >
             <a
               :href="`#${link.id}`"
-              class="block text-sm scrollactive-item transition-padding ease-in-out duration-300 hover:pl-1"
+              class="block text-sm scrollactive-item transition-padding ease-in-out duration-300 hover:ps-1"
               :class="{
-                'py-2': link.depth === 2,
-                'ml-2 pb-2': link.depth === 3,
-                'ml-3 pb-2': link.depth === 4,
-                'ml-4 pb-2': link.depth === 5,
-                'ml-5 pb-2': link.depth === 6
+                'py-2 font-semibold': link.depth === 2,
+                'ms-2 pb-2': link.depth === 3,
+                'ms-3 pb-2': link.depth === 4,
+                'ms-4 pb-2': link.depth === 5,
+                'ms-5 pb-2': link.depth === 6
               }"
             >{{ link.text }}</a>
           </li>
diff --git a/node_modules/@nuxt/content-theme-docs/src/components/global/base/List.vue b/node_modules/@nuxt/content-theme-docs/src/components/global/base/List.vue
index 91d7a09..294540e 100644
--- a/node_modules/@nuxt/content-theme-docs/src/components/global/base/List.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/components/global/base/List.vue
@@ -1,7 +1,7 @@
 <template>
   <div>
     <div v-for="(item, i) in items" :key="i" class="mt-3 flex">
-      <span :class="`list-${type}`" class="mt-px mr-3 flex-shrink-0">
+      <span :class="`list-${type}`" class="mt-px me-3 flex-shrink-0">
         <component :is="iconName" class="h-6 w-6" />
       </span>
       {{ item }}
diff --git a/node_modules/@nuxt/content-theme-docs/src/i18n/ar-AR.js b/node_modules/@nuxt/content-theme-docs/src/i18n/ar-AR.js
new file mode 100644
index 0000000..c8301c4
--- /dev/null
+++ b/node_modules/@nuxt/content-theme-docs/src/i18n/ar-AR.js
@@ -0,0 +1,12 @@
+export default {
+  search: {
+    placeholder: "ابحث في المستندات (اضغط على" / "للتركيز)"
+  },
+  toc: {
+    title: 'على هذه الصفحة'
+  },
+  article: {
+    github: "تحرير هذه الصفحة على github",
+    updatedAt: 'تم التحديث في'
+  }
+}
diff --git a/node_modules/@nuxt/content-theme-docs/src/layouts/default.vue b/node_modules/@nuxt/content-theme-docs/src/layouts/default.vue
index 56e9a35..0c14bdb 100644
--- a/node_modules/@nuxt/content-theme-docs/src/layouts/default.vue
+++ b/node_modules/@nuxt/content-theme-docs/src/layouts/default.vue
@@ -1,5 +1,5 @@
 <template>
-  <div class="pt-16">
+  <div class="pt-16" :dir="this.$i18n.locale == 'ar' ? 'rtl' : 'ltr'">
     <AppHeader />
 
     <main class="container mx-auto px-4 lg:px-8">
